kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/schema/parse.kt

```diff
@@ -73,26 +70,28 @@ internal object JsonSchemaArraySerializer : KSerializer<JsonSchema.JsonArray> {
    override fun deserialize(decoder: Decoder): JsonSchema.JsonArray =
       decoder.decodeStructure(descriptor) {
          var matcher: Matcher<Sequence<JsonNode>>? = null
-         var minItems = 1
-         var maxItems = Int.MAX_VALUE
-         val elementType = decodeSerializableElement(descriptor, 4, SchemaDeserializer, null)
+         val minItems = kotlin.runCatching { decodeIntElement(descriptor, 1) }.getOrDefault(1)
```

Comment:
```suggestion
         val minItems = runCatching { decodeIntElement(descriptor, 1) }.getOrDefault(1)
```
kotlin prefix shouldnt be needed